scripts/gen3text.py

```diff
@@ -431,27 +431,28 @@ def utf8ToRSText(t, region = ""):
 	if region == "DE":
 		chars['“'] = '\xB2'
 
-	characters = []
-	char = ""
-	while len(t):
-		if ord(t[0]) >= 0xF0:
-			char += t[0:4]
-			t = t[4:]
-		elif ord(t[0]) >= 0xE0:
-			char += t[0:3]
-			t = t[3:]
-		elif ord(t[0]) >= 0xC0:
-			char += t[0:2]
-			t = t[2:]
-		else:
-			char += t[0:1]
-			t = t[1:]
-		if char != "\\" and char != "\\v" and (char[0:2] != "\\{" or char[-1] == "}"):
-			characters.append(char)
-			char = ""
+#	characters = []
+#	char = ""
+#	while len(t):
+#		if ord(t[0]) >= 0xF0:
```

Comment:
I can't tell if this is necessary or not. It really feels like a hack that was inserted to handle py2's poor utf8 handling and the code works without it. 

Leaning towards just deleting it then restoring in some capacity if it ever causes an issue in the future

At the very least, keeping it in breaks things in py3